pkg/hintrunner/zero/zerohint_dictionaries.go

```diff
@@ -10,6 +10,55 @@ import (
 	f "github.com/consensys/gnark-crypto/ecc/stark-curve/fp"
 )
 
+// DefaultDictNew hint creates a new dictionary with a default value
+//
+// `newDefaultDictNewHint` takes 1 operander as argument
+//   - `default_value` variable will be the default value
+//     returned for keys not present in the dictionary
+func newDefaultDictNewHint(defaultValue hinter.ResOperander) hinter.Hinter {
+	return &GenericZeroHinter{
+		Name: "DefaultDictNew",
+		Op: func(vm *VM.VirtualMachine, ctx *hinter.HintRunnerContext) error {
+			//> if '__dict_manager' not in globals():
+			//> 	from starkware.cairo.common.dict import DictManager
+			//> 	__dict_manager = DictManager()
+			//>
+			//> memory[ap] = __dict_manager.new_default_dict(segments, ids.default_value)
+
+			//> if '__dict_manager' not in globals():
+			//> 	from starkware.cairo.common.dict import DictManager
+			//> 	__dict_manager = DictManager()
+			dictionaryManager, ok := ctx.ScopeManager.GetZeroDictionaryManager()
```

Comment:
usually, we skip a line between the comment with the python hint. and the beginning of the code, but doesnt matter